webapp/src/Controller/SecurityController.php

```diff
@@ -132,7 +132,9 @@ public function registerAction(
             $plainPass = $registration_form->get('plainPassword')->getData();
             $password  = $passwordEncoder->encodePassword($user, $plainPass);
             $user->setPassword($password);
-            $user->setName($user->getUsername());
+            if ($user->getName() == null) {
```

Comment:
Please use strict comparison with `===`

Reply:
Thanks! Will make a patch in a minute.
(sorry, not an expert in php)

Reply:
@thijskh just curious: when it makes a different in this situation? I just read that 0 == null returns true and 0 === null returns false, but in our case the name cannot be an integer, can it?

Reply:
So indeed it might not matter much in this specific case. But the policy is to be as type strict as possible in our code. So you're not dependent on some judgement that for this case it "would not matter". Just do it always.

Reply:
I see. Actually now I see now that there is a function empty(..) to check if the field is empty (e.g., in validation constraints). Should probably use this instead for the sake of consistency.